### PR TITLE
Release 1.1.10

### PR DESCRIPTION
**Pull request by cyphar**
Update of #4092 to include the #4103 fix.

<hr>

## [1.1.10] - 2023-10-30

> Śruba, przykręcona we śnie, nie zmieni sytuacji, jaka panuje na jawie.

### Added

* Support for `hugetlb.<pagesize>.rsvd` limiting and accounting. Fixes the
  issue of postres failing when hugepage limits are set. (#3859, #4077)

### Fixed

* Fixed permissions of a newly created directories to not depend on the value
  of umask in tmpcopyup feature implementation. (#3991, #4060)
* libcontainer: cgroup v1 GetStats now ignores missing `kmem.limit_in_bytes`
  (fixes the compatibility with Linux kernel 6.1+). (#4028)
* Fix a semi-arbitrary cgroup write bug when given a malicious hugetlb
  configuration. This issue is not a security issue because it requires a
  malicious `config.json`, which is outside of our threat model. (#4103)
* Various CI fixes. (#4081, #4055)

### CODE REVIEW

**Review by AkihiroSuda — APPROVED** [review]

**Comment by thaJeztah** [comment]
Do we still need to refer to the CVE, or should we remove that? (We can remove the CVE from the advisory, and still link to (and publish) the advisory, but we could decide not to attach a CVE to it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 ## [Unreleased 1.1.z]
 
+## [1.1.10] - 2023-10-30
+
+> Śruba, przykręcona we śnie, nie zmieni sytuacji, jaka panuje na jawie.
+
+### Security
+
+* Fix a semi-arbitrary cgroup write bug when given a malicious hugetlb
+  configuration. As discussed in the [security advisory][cve-2023-45804], our
```
Comment (thaJeztah) [inline CHANGELOG.md:16]: the CVE has been revoked, so instead we can link to the advisory for details (which can be viewed after we publish it)

Comment (kolyshkin) [reply CHANGELOG.md:16]: In fact, the text does not mention CVE, it is just a label.

Comment (cyphar) [reply CHANGELOG.md]: I opened a dummy issue (#4103) for the advisory and have added a reference to it.

**Review by lifubang — APPROVED** [review]

**Pull request merged**